doc/rules/basic/trailing_comma_in_multiline.rst

```diff
@@ -124,7 +124,11 @@ The rule is part of the following rule sets:
   ``['after_heredoc' => true]``
 
 @PhpCsFixer
-  Using the `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_ rule set will enable the ``trailing_comma_in_multiline`` rule with the default config.
+  Using the `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_ rule set will enable the ``trailing_comma_in_multiline`` rule with the config below:
+
+  ``['elements' => ['array_destructuring', 'arrays', 'match']]``
 
 @Symfony
-  Using the `@Symfony <./../../ruleSets/Symfony.rst>`_ rule set will enable the ``trailing_comma_in_multiline`` rule with the default config.
+  Using the `@Symfony <./../../ruleSets/Symfony.rst>`_ rule set will enable the ``trailing_comma_in_multiline`` rule with the config below:
+
+  ``['elements' => ['array_destructuring', 'arrays', 'match']]``
```

Comment:
The default value (as was previously used) is `['arrays']`.

Reply:
This is a new rule, as such there is no "previously used"?

Reply:
> This is a new rule

Not sure I follow you. What makes you think the rule is new?

Anyway, the `@Symfony` ruleset isn't new so it shouldn't change. But actually Symfony might be OK with this? @nicolas-grekas What do you think?

Reply:
`trailing_comma_in_multiline` rule is already on master, so it is not a new rule indeed

yet, the configuration of it for Sf ruleset changed - not sure if it's welcome by Sf community, did we checked if repo follows this new rule already, or checked with maintainers?

Reply:
I confused this PR with another, sorry about that.
The changes on SF 5.4 would look like this:
https://github.com/SpacePossum/symfony/pull/2/files